go.mod

```diff
@@ -18,14 +18,15 @@ require (
 	github.com/google/uuid v1.2.0
 	github.com/klauspost/cpuid/v2 v2.0.9
 	github.com/kr/pretty v0.2.1 // indirect
-	github.com/linuxboot/fiano v1.0.4-0.20211220133044-7de4aa4d8e3d
+	github.com/linuxboot/fiano v1.0.4-0.20220110153808-38194836b6f1
```

Comment:
I'm now worried about the fact that we're not on a major version yet for `fiano` and how future updates may make upgrading this/maintaining it painful. 

That's future me's problem I guess, but something to keep in mind

Reply:
Sorry, cannot focus my mind. Could you rephrase, please?

Reply:
If we release a breaking API change in `fiano` for the parts that `converged-security-suite` depends on, unless we upgrade the major version of `fiano`, resolving dependencies will not be fun.

Reply:
No, I initially created these packages in fiano with updated API (to be more consistent with other fiano modules in terms of how they are used). There was no breaking change inside of fiano. The breaking change comes from switching itself (from CSS's version of the packages to fiano's version of the packages).